### DISCUSSION TITLE
Request for Comment - Windows Update

### DISCUSSION THREAD

**Discussion by HotCakeX** [HotCakeX/Harden-Windows-Security#11] [Q&A] [answered]
Should Windows Update policies be added to the Security Baseline X? if so, which one(s)?

https://techcommunity.microsoft.com/t5/windows-it-pro-blog/the-windows-update-policies-you-should-set-and-why/ba-p/3270914

**Comment by innovatodev** [comment, marked as answer]
It should be. Microsoft fixes severe security vulnerabilities almost every month ^^ an outdated windows cannot be hardened in my opinion.

Maybe you could offer the 3 policies as an option, these 3 : 
Single-user or personal devices
Multi-user devices
Education devices
are all secure and all have their advantage and incoveniants, explain rapidly wich one do what and offer it as an option ^^ or at least the first 2, either single or multi device ^^
